README.md

```diff
@@ -140,6 +140,9 @@ configuration:
     modules:
       source: "%(component)s.git#%(stream)s"
       destination: "%(component)s.git#%(stream)s-fluff-42.0.0-alpha"
+      rpms:
+        source: "%(component)s.git#%(stream)s"
```

Comment:
I'd drop the %(stream)s here; if anything, it could be %(ref)s which we also discussed but since we're looking for the modulemd-provided ref in the entire source repo, I'd leave it blank like in the defaults/rpms section.

Reply:
Thanks! I'll make that revision, and also correct the description of the format strings below to make it clear that `%(name)` and `%(ref)` can only be used in the module sub-component defaults.